compose.yml

```diff
@@ -44,6 +44,7 @@ services:
         soft: 65535
         hard: 65535
     environment:
+      - TMPDIR=/var/lib/plausible/tmp
```

Comment:
TMPDIR is chosen here since it covers both [System.tmp_dir](https://github.com/elixir-lang/elixir/blob/9d5af3fe320fcdfce36af7f0b76ee573896c9adf/lib/elixir/lib/system.ex#L383-L386) and [Plug.Upload.random_file](https://github.com/elixir-plug/plug/blob/e11e5c4cb2ceed52914af460d3d9bb07a84fc8eb/lib/plug/upload.ex#L43)